.github/workflows/multicluster.yaml

```diff
@@ -0,0 +1,244 @@
+name: Multicluster / Cluster mesh (ci-multicluster)
+
+on:
+  issue_comment:
+    types:
+      - created
+  push:
+    branches:
+      - master
+  ### FOR TESTING PURPOSES
+  # pull_request:
+  #  types:
+  #    - "labeled"
+  ###
+
+env:
+  clusterName1: cilium-cli-ci-${{ github.run_id }}-multicluster-1
+  clusterName2: cilium-cli-ci-${{ github.run_id }}-multicluster-2
+  zone: us-west2-a
+  check_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
+
+jobs:
+  installation-and-connectivity:
+    if: |
+      (github.event.issue.pull_request && startsWith(github.event.comment.body, 'ci-multicluster')) ||
```

Comment:
I didn't add multicluster testing to `test-me-please`, should I?

Reply:
Yes, let's enable everything and disable if it turns out too much.